### DISCUSSION TITLE
Test ijv and egzumer

### DISCUSSION THREAD

**Discussion by pietss** [egzumer/uv-k5-firmware-custom#224] [General]
A test; from what I understand, the author made a comparison, on that frequency, between ijv 2.3r5 e 2.4 and egzumer, and the one he receives better is egzumer
https://www.facebook.com/coupe.fiat.9/videos/1079888489850838/?idorvanity=1432752317481975

**Comment by ErikS-web** [comment]
Thanks, but something is missing from the test....

Then swap the firmware on the radios to test whether it stays that way.
Could be the hardware.

**Comment by pietss** [comment]
I'll summarize what they say. The person who did the test says he will try two more radios tomorrow; a person suggests him to do the test with the same radio (maybe it's you? ;-)) and the author replies that he tested on 3 radios. Perhaps we need to wait for the new tests that he will carry out to be certain (on that frequency).

**Comment by pietss** [comment]
The experimenter has disappeared, or perhaps he doesn't want to divulge this discovery too much, or perhaps he has received threats ;-), the world of UV-K5 radio users is picturesque, let's not lose heart, one day the truth will emerge.
Beyond the irony, the success of this radio is interesting in many ways, not excluding that of the not rare almost deliriums in the various forums.

**Comment by iu0azb** [comment]
Hi all let me get back on this discussion :)
After hopping between egzumer and ijv firmware on a daily basis trying to figure out which one I preferred.... I decided to buy a second k5 to have more peace of mind, although I'll have to choose one day :)

From my practical experience, even reversing firmware and radio, I find the IJV firmware slightly better in reception. 
The ability to adjust gain manually is very usefull using external antennas and the converter.  Also very handy is the ability to select the bandwidth with a button and speaking of filtering, currently IJV has implemented 4, RF mixing and audio filtering.
 	N- (1.7kHz 6.25kHz)
 	N (3kHz 12.5kHz)
 	W (6kHz 25kHz)
 	W+ (9kHz 25kHz)
What I really hate of it is that only bin"" file is released with not very detailed notes.  So no way to know changes or modify and recompile it.   The DSB reception quality is better that other firmwares, about the famouse DSB trasmission, I'm testing it successfully, just underline that it's not SSB but DSB. I've already enjoied several QSO in VHF while waling during trekking sessions :)

Ezgumer software, on the other hand, besides the super bandscope that is indispensable, is more'customizable from menu, Backlight is adjustable, and the integration with chirp offers incredible convenience. Lastly, the collaboration of an opensource adds positive grades to the project :)

Hope one day to find the final decision getting just one radio with me :)

**Comment by VK3EAN** [comment]


**Reply by iu0azb** [reply]
Hi Erwin, 
on IJV I'm using the original chirp uvk5 module.. it seem to work properly.

**Comment by AndrewTXJ** [comment]
is there a link for IJV? i could test it out.

**Reply by ErikS-web** [reply]
PSE discuss other firmware issues (like IJV) on other medium than here.

If you swap firmware that is not in memory-mapping equal (like IJV) you wil have unespected problems

Do a full reset or restore eeprom .... or use a separate radio.

**Reply by iu0azb** [reply]
https://www.universirius.com/preppers/quansheng-uv-k5-manuale-del-firmware-ijv/

You just need to save the calibration and configuration  files first.

The only issue occurred ups today on 2.6 release, not sure why ;)
